HW0/P5/P5.txt

```diff
@@ -1 +1,54 @@
 # Your answers here.
+
+1. It would take 255 or 256 seconds (depending on whether or not you count
+adding the first value to 0 as a second or not) 
+
+2. We have that 256 / 8 = 32, so in just 32 seconds, each of the 8 workers 
+will have a number that represents the number of pennies they counted
+indivudually. Then, assuming that communication takes no time, in 1 second we 
+could take the 8 numbers and make them into 4, by assigning 4 people 2 numbers 
+to add each. Then, in another second, we could turn 4 numbers into 2, and 
+then 2 numbers into 1. Therefore, in total, this process takes 35 seconds. 
+
```

Comment:
Again you assumed that it takes N seconds to count N bags. But it should be N-1 seconds to count N bags by my understand. Hence the answer to this should be 34 seconds. But the process you are using is perfectly right.